DQM/TrackingMonitor/src/TrackAnalyzer.cc

```diff
@@ -1583,6 +1599,78 @@ void TrackAnalyzer::bookHistosForState(std::string sname, DQMStore::IBooker& ibo
+  tkmes.TrackPtLoose->setAxisTitle("Number of Loose Tracks", 2);
+
+  histname = "Quality_";
+  tkmes.Quality = ibooker.book1D(histname + CategoryName, histname + CategoryName, 3, 0., 3.);
+  tkmes.Quality->setAxisTitle("Track quality", 1);
+  tkmes.Quality->setAxisTitle("Number of Tracks", 2);
+
+  for (size_t ibin = 0; ibin < 3; ibin++) {
+    tkmes.Quality->setBinLabel(ibin + 1, reco::TrackBase::qualityNames[ibin]);
+  }
+
+  histname = "TrackPtZone1_" + histTag;
+  tkmes.TrackPtZone1 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone1->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone1->setAxisTitle("Number of Tracks", 2);
+
+  histname = "TrackPtZone2_" + histTag;
+  tkmes.TrackPtZone2 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone2->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone2->setAxisTitle("Number of Tracks", 2);
+
+  histname = "TrackPtZone3_" + histTag;
+  tkmes.TrackPtZone3 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone3->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone3->setAxisTitle("Number of Tracks", 2);
+
+  histname = "TrackPtZone4_" + histTag;
+  tkmes.TrackPtZone4 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone4->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone4->setAxisTitle("Number of Tracks", 2);
+
+  histname = "TrackPtZone5_" + histTag;
+  tkmes.TrackPtZone5 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone5->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone5->setAxisTitle("Number of Tracks", 2);
+
+  histname = "TrackPtZone6_" + histTag;
+  tkmes.TrackPtZone6 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone6->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone6->setAxisTitle("Number of Tracks", 2);
+
+  histname = "TrackPtZone7_" + histTag;
+  tkmes.TrackPtZone7 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone7->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone7->setAxisTitle("Number of Tracks", 2);
+
+  histname = "TrackPtZone8_" + histTag;
+  tkmes.TrackPtZone8 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.TrackPtZone8->setAxisTitle("Track p_{T} (GeV/c)", 1);
+  tkmes.TrackPtZone8->setAxisTitle("Number of Tracks", 2);
+
+  histname = "Ratio_byFolding_" + histTag;
+  tkmes.Ratio_byFolding = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.Ratio_byFolding->setAxisTitle("Track p_{T} (GeV/c)", 1);
+
+  histname = "Ratio_byFolding2_" + histTag;
+  tkmes.Ratio_byFolding2 = ibooker.book1D(histname, histname, TrackPtBin, TrackPtMin, TrackPtMax);
+  tkmes.Ratio_byFolding2->setAxisTitle("Track p_{T} (GeV/c)", 1);
+
```

Comment:
@hbecerri please set non meaningful name in the title of the plots (second histname in book1D), otherwise it will be difficult to know what ZoneX stands for. Thanks

Reply:
Hello @jfernan2, I did the modifications. I tried to be more specific.